html/includes/functions.inc.php

```diff
@@ -1133,6 +1188,37 @@ function alert_details($details)
             $fallback = false;
         }
 
+        if ($tmp_alerts['sensor_id']) {
+            $details = "Current Value: " . $tmp_alerts['sensor_current'] . " (" . $tmp_alerts['sensor_class'] . ")<br>  ";
+            if ($tmp_alerts['sensor_limit_low']) {
+                $details_a[] = "low: " . $tmp_alerts['sensor_limit_low'];
+            }
+            if ($tmp_alerts['sensor_limit_low_warn']) {
+                $details_a[]= "low_warn: " . $tmp_alerts['sensor_limit_low_warn'];
+            }
+            if ($tmp_alerts['sensor_limit_warn']) {
+                $details_a[]= "high_warn: " . $tmp_alerts['sensor_limit_warn'];
+            }
+            if ($tmp_alerts['sensor_limit']) {
+                $details_a[]= "high: " . $tmp_alerts['sensor_limit'];
+            }
+            $details .= implode(', ', $details_a);
```

Comment:
Scrutinizer: "The variable ``$details_a`` does not seem to be defined for all execution paths leading up to this point."

Reply:
done